routes/api/promotions.js

```diff
@@ -37,6 +55,23 @@ promotionsRouter.get('/:promotionId', finder.findById);
 
 
 
+promotionsRouter.post('/public-ad/click/:promotionId', async (req, res) => {
+    if (await Promotion.findOne()) {
+       const promotion = await Promotion.findByPk(req.params.promotionId);
+       const cont = 1;
+       let valorActual = promotion.dataValues.clickedQuantities;
+       let valorNuevo = valorActual + cont;
+       await promotion.update({ clickedQuantities: valorNuevo });
+       res.status(200).json({ success: true });
```

Comment:
Falta try y catch por si las moscas
```suggestion
       await promotion.update({ clickedQuantities: valorNuevo });
       res.status(200).json({ success: true });
```
Otra cosa, esto no habia cambiado? para ser una redireccion 302?? 🤔

Reply:
xd